### DISCUSSION TITLE
Show issues in an Epic

### DISCUSSION THREAD

**Discussion by winslowb** [ankitpokhrel/jira-cli#185] [Q&A] [answered]
Hi,

I'm trying to view issues in an epic. I receive and error that I need to edit the issue and assign a parent but I've not been able to successfully add the issue to the epic. 

I've tried to 
* jira epic add GPE-134 GPE-171 (134 is the epic and 171 is the issue)
* jira issue edit GPE-171 `"parent":{"key":"GPE-134"}`
![image](https://user-images.githubusercontent.com/71040996/140659379-104bbdf8-8460-447b-9d7d-2069151485a9.png)

**Comment by ankitpokhrel** [comment]
Looks like epics are handled differently in a next gen project 😭

I might need to make some changes to support this. Meanwhile, I am going to convert this discussion to an issue and hopefully make it work soon.

Thanks for reporting @winslowb 

See: https://github.com/ankitpokhrel/jira-cli/issues/186

**Comment by ankitpokhrel** [comment, marked as answer]
@winslowb The issue with next-gen project is fixed in [v0.2.0](https://github.com/ankitpokhrel/jira-cli/releases/tag/v0.2.0).

Please feel free to report if you encounter any issue with the new release. Thanks!
